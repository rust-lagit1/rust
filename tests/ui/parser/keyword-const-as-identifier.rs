fn main() {
    let const = "foo";
    //~^ ERROR expected `{`, found `=`
    //~| ERROR inline-const in pattern position is experimental [E0658]
}
