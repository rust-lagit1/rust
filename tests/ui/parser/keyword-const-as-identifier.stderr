error: expected `{`, found `=`
  --> $DIR/keyword-const-as-identifier.rs:2:15
   |
LL |     let const = "foo";
   |               ^ expected `{`

error[E0658]: inline-const in pattern position is experimental
  --> $DIR/keyword-const-as-identifier.rs:2:9
   |
LL |     let const = "foo";
   |         ^^^^^
   |
   = note: see issue #76001 <https://github.com/rust-lang/rust/issues/76001> for more information
   = help: add `#![feature(inline_const_pat)]` to the crate attributes to enable
   = note: this compiler was built on YYYY-MM-DD; consider upgrading it if it is out of date

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0658`.
